src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -488,22 +494,140 @@ private void testFileWithoutInfoLineInHeader(final String name, final Class<? ex
+
+    // If two separate --select commandlines are used e.g. --select "'AF > 0.05'" --select "'MQRankSum > 3'"
+    // then the booleans should be combined with the logical-or
+    @Test
+    public void testMultipleJexls() {
+        // multiple INFO jexls (separate test, combine genotype and info jexls)
+        beforeJexlTest();
+        final GATKPath testVcf = new GATKPath(getToolTestDataDir() + "hgdb_sv_multi_sample_mini.vcf");
+        // Is File acceptable here? Path? GATKPath?
+        final File testOutput = createTempFile("multiple_jexl_test", "vcf");
+
+        // START Chris/Louis/David
+        // Attempt 1, does not work
+//        final ArgumentsBuilder args1 = new ArgumentsBuilder()
+//                .add("V", testVcf.toString())
+//                .add("O", testOutput.getAbsolutePath())
+//                .add("--select", "'AC > 1'")
+//                .add("--select", "'AF > 0'");
+//        // .add("select", "'GQ > 0'"); // tsato: this does not work for some reason...
+//        runCommandLine(args1, SelectVariants.class.getSimpleName());
+
+        // Attempt 2 (remove single quotes), does not work
+//        final ArgumentsBuilder args2 = new ArgumentsBuilder()
+//                .add("V", testVcf.toString())
+//                .add("O", testOutput.getAbsolutePath())
+//                .add("--select", "AC > 1")
+//                .add("--select", "AF > 0");
+//        // .add("select", "'GQ > 0'"); // tsato: this does not work for some reason...
+//        runCommandLine(args2, SelectVariants.class.getSimpleName());
+
+        // Atempt 3 (remove spaces). commandline parser lets this through but not the JEXL parser
+//        final ArgumentsBuilder args3 = new ArgumentsBuilder()
+//                .add("V", testVcf.toString())
+//                .add("O", testOutput.getAbsolutePath())
+//                .addRaw("--select 'AC>1'")
+//                .addRaw("--select 'AF>0'");
+//                // .add("select", "'GQ > 0'"); // tsato: this does not work for some reason...
+//        runCommandLine(args3, SelectVariants.class.getSimpleName());
+
+        // Attempt 4 (give a list to runCommandLine), works
+//        runCommandLine(Arrays.asList("-V", testVcf.toString(),
+//                        "-O", testOutput.getAbsolutePath(),
+//                        "--select","AC > 1"),
+//                SelectVariants.class.getSimpleName());
+
+        // Attempt 5, works!
+//        runCommandLine(Arrays.asList("-V", testVcf.toString(),
+//                        "-O", testOutput.getAbsolutePath(),
+//                        "--select", "ALGORITHMS == 'depth'"),
+//                SelectVariants.class.getSimpleName());
+
+        // Attempt 6, works!
+        runCommandLine(Arrays.asList("-V", testVcf.toString(),
+                        "-O", testOutput.getAbsolutePath(),
+                        "--select", "SVTYPE == 'DEL'"),
+                SelectVariants.class.getSimpleName());
+        // END Chris/Louis/David
```

Comment:
@droazen @lbergelson @cmnbroad this block of code has the command line parsing issue I mentioned. My work around is Attempt 5 and 6, where I feed `runCommandLine` a list of arguments rather than an argument builder. Is that good enough? Otherwise I suspect we need to update `ArgumentsBuilder` to support JEXL command lines.